src/main/java/io/fabric8/maven/docker/config/BuildImageConfiguration.java

```diff
@@ -330,7 +330,16 @@ public File getAbsoluteContextDirPath(MojoParameters mojoParams) {
     }
 
     public File getAbsoluteDockerFilePath(MojoParameters mojoParams) {
-        return EnvUtil.prepareAbsoluteSourceDirPath(mojoParams, getDockerFile().getPath());
+        File absoluteDockerFilePath = EnvUtil.prepareAbsoluteSourceDirPath(mojoParams, getDockerFile().getPath());
```

Comment:
Is it okay to update `contextDir` in a getter method like this??

Or shall we change initAndValidate method in order to handle Dockerfile early also, but not sure that has access to MojoParams